reflectapi-demo/reflectapi.json

```diff
@@ -9,7 +9,8 @@
       "serialization": [
         "json"
       ],
-      "readonly": true
+      "readonly": true,
```

Comment:
can we generalize this and introduce tags field as a vector of strings? Then we would be able to include / exclude routes / types and fields by tags by providing --include-tags and --exclude-tags arguments to codegen tools.

Reply:
Mostly done, do you expect this to apply to rust/ts too?

Reply:
possibly eventually, most likely TS, C# and PHP. I can imagine we can offer generated SDKs to customers by including only features which are tagged for that customer (or group of customers). We have got the plans to narrow down the scope of APIs available significantly

Reply:
but not high priority at the moment